### PR TITLE
hub: add hub pr usage for checkout someone created pull-request

### PR DESCRIPTION
**Pull request by conao3**
Hi! Thanks for such a great project!

I add `hub pr` usege for checkout someone created pull-request.

Cited from `hub` official readme https://hub.github.com/#maintainer

best,

- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by conao3** [comment]
I need to add https://hub.github.com/#maintainer to usage?
Since the other examples in `hub` do not contain url, adding only this one seems disproportionate.

File: pages/common/hub.md [hunk 1]
```diff
@@ -24,6 +24,10 @@
 
 `hub pull-request --no-edit`
 
+- Create a new branch with the contents of the pull request and checkout it for review:
```
Comment (waldyrious) [inline pages/common/hub.md]: How about this?

```suggestion
- Create a new branch with the contents of a pull request and switch to it:
```

**Comment by waldyrious** [comment]
Huh. That's strange, I was under the impression that this subcommand was already present in this page. Thanks for adding it! Please look at the comment I left inline.

> I need to add hub.github.com/#maintainer to usage?

I'm not sure I understand your question. There's already an URL on this page. Are you suggesting we should change it? Or add the new one somewhere else on the page?

File: pages/common/hub.md (hunk 1 above)
Comment (conao3) [reply pages/common/hub.md]: Good! thanks.

**Comment by conao3** [comment]
Sorry, I misunderstand tldr maintainer required me to one URL per example. But one URL per page is true.

**Comment by conao3** [comment]
Updated!

**Comment by sbrl** [comment]
Force-merging because travis seems to have gotten itself stuck. I manually checked the syntax and it's ok :-)

**Comment by conao3** [comment]
OK, thanks! I'll keep contribute to this project 🎉

**Pull request merged**